docs/CSBR.md

```diff
@@ -503,19 +503,16 @@ If the CA decides that the revocation will have an unreasonable impact on its cu
 
 The CA MUST revoke a Code Signing Certificate within one (1) business day if the Subscriber requests in writing that the CA revoke the Certificate or notifies the CA that the original certificate request was not authorized and does not retroactively grant authorization.
 
-#### 4.9.1.3 Revocation Based on Reported or Detected Compromise or Use in Malware
+#### 4.9.1.3 Revocation Based on Reported or Detected Compromise or Use in Suspect Code
 
-For all incidents involving malware, CAs SHALL revoke the Code Signing Certificate in accordance with and within the following maximum timeframes. Nothing herein prohibits a CA from revoking a Code Signing Certificate prior to these timeframes.
+Except for cases that fall under Section 4.9.1.1, if, while investigating a problem report, the CA determines the Subscriber's Private Key is likely compromised or being used for Suspect Code, the CA SHALL revoke the corresponding Code Signing Certificate in accordance with and within the following maximum time frames. Nothing herein prohibits a CA from revoking a Code Signing Certificate prior to these time frames.
 
-1.  The CA MUST contact the software publisher within one (1) business day after the CA is made aware of the incident.
-2.  The CA MUST determine the volume of relying parties that are impacted (e.g., based on OCSP logs) within 72 hours after being made aware of the incident.
-3.  The CA MUST request the software publisher send an acknowledgement to the CA within 72 hours of receipt of the request.
-  a.  If the publisher responds within 72 hours, the CA and publisher MUST determine a "reasonable date" to revoke the certificate based on discussions with the CA.
-  b.  If CA does not receive a response, the CA must notify the publisher that the CA will revoke in 7 days if no further response is received.
-    i.  If the publisher responds within 7 days, the CA and the publisher will determine a "reasonable date" to revoke the certificate based on discussion with the CA.
-    ii.  If no response is received after 7 days, the CA must revoke the certificate except if the CA has documented proof (e.g., OCSP logs) that the revocation will cause significant impact to the general public.
+1.  The CA MUST contact the Subscriber within 24 hours after the CA received the initial problem report.
+2.  The CA MUST request the Subscriber send an acknowledgement to the CA within 72 hours of receipt of the request.
+   a.  If the Subscriber responds within 72 hours, the CA and Subscriber MAY determine a "reasonable date" to revoke the certificate. The revocation date MUST NOT be more than 7 calendar days after the incident was discovered.

```

Comment:
For Consistency and accuracy, please consider updating to:

- "The revocation date MUST NOT be more than 7 calendar days after the CA received the initial problem report or discovered the incident."

Reply:
I have a concern with the phrase "the incident was discovered", as it's fundamentally the same thing as "the CA is made aware of the incident". The problem with this language is that it's subjective and thus difficult to fulfill.

For example, it's unclear whether a researcher's email to a random CA sales rep is considered "discovery" of an incident and is subject to the timelines defined in this ballot. For this reason, I think the previous language (omitting the "or discovered the incident") provides clarity in regard to the CA's obligation to timely handle such incidents.

Reply:
@CBonnell  wouldn't the same apply though for "received the initial problem report". One could read an email sent to a sales rep containing an issue as a problem report, just not to the address specified by the CA?

Reply:
Perhaps not explicit, but I read "problem report" as "Certificate Problem Report", which has well defined obligations for disclosure of the reporting address and subsequent handling of the report.

Reply:
@CBonnell I see. Trying to find some common ground here where the same process is followed when either the CA discovers the incident themselves, or get a report.

Trying to come up with a solution that will basically invoke the section if the CAs abuse or problem reporting department discovered the incident. However since there's no defined CA department name for this, I'm not sure that would make it any clearer.

Thoughts anyone?

Reply:
@chrisbn I think that makes sense and I like that approach. There are two occurrences of "initial problem report" in the section, both could be updates.
@CBonnell  your take on this?

Reply:
As @chrisbn suggested, I think replacing "initial problem report" with "Certificate Problem Report" improves clarity.